test/unit/reporters/CLIReporter-test.js

```diff
@@ -15,13 +15,17 @@ describe('CLIReporter', () => {
   let test = {};
 
   before(() => {
-    loggerStub.transports.console.silent = true;
-    reporterOutputLoggerStub.transports.console.silent = true;
+    [loggerStub, reporterOutputLoggerStub].forEach((logger) => {
+      logger.configure({ silent: true });
+    })
   });
 
   after(() => {
-    loggerStub.transports.console.silent = false;
-    reporterOutputLoggerStub.transports.console.silent = false;
+    // Is this really a good idea to mutate logger instances
+    // vs creating logger instances for tests using the same factory function?
```

Comment:
> Is this really a good idea to mutate logger instances

Nope 😄 But as per my previous comment, currently the logging isn't done in a way it would be easy to test. It is planned, but if you want to keep this PR scoped to Winston upgrade, you should restrain yourself from trying to solve this problem as well. We can (should!) do it as one of the next steps.

We can limit the mutations though by ommitting silencing of the logging in tests, which is not for the purpose of the test itself, but only for the purpose of not having the `npm test` output cluttered (like in this case).

Reply:
Thanks for clarifying. I will remove this mutation and see the test output.

Let's keep this pull request scoped to prevent the complexity of the changes. We can refactor the usage of logging later, I'd be glad to.